code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/local/LocalAuthPasswordValidator.java

```diff
@@ -0,0 +1,55 @@
+package io.cattle.platform.iaas.api.auth.integration.local;
+
+import io.cattle.platform.archaius.util.ArchaiusUtil;
+import io.cattle.platform.json.JsonMapper;
+import io.github.ibuildthecloud.gdapi.exception.ClientVisibleException;
+import io.github.ibuildthecloud.gdapi.util.ResponseCodes;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.entity.ContentType;
+
+import com.netflix.config.DynamicStringProperty;
+
+public class LocalAuthPasswordValidator {
+
+    public static final DynamicStringProperty AUTH_VALIDATE_URL = ArchaiusUtil.getString("api.auth.local.validate.url");
+
+    public static void validatePassword(String password, JsonMapper jsonMapper) {
```

Comment:
It wouldn't let me inject and I'd have to make it static to be used by `validatePassword`.